src/COLLADA2GLTFWriter.cpp

```diff
@@ -665,6 +665,17 @@ bool COLLADA2GLTF::Writer::writeMesh(const COLLADAFW::Mesh* colladaMesh) {
 				buildIndices.push_back(buildIndices[end]);
 				buildIndices.push_back(buildIndices[startFace]);
 			}
+#ifdef USE_DRACO
+			if (_options->dracoCompression ) {
+				// TODO: Support other modes.
```

Comment:
Instead of TODO, just a comment that says something like "this currently only supports triangles" would be fine